packages/fxa-payments-server/src/components/PlanDetails/index.tsx

```diff
@@ -52,6 +52,8 @@ export const PlanDetails = ({
     interval_count
   );
 
+  console.log(config.featureFlags.subscriptionCoupons)
```

Comment:
small nit in case you get to bianca's before this is merged, can we remove the console.log statement to keep the console clean? otherwise, non-blocker.